jbpm-human-task/jbpm-human-task-jpa/src/main/java/org/jbpm/services/task/impl/model/ContentDataImpl.java

```diff
@@ -51,6 +68,53 @@ public void setType(String type) {
 	public void setContent(byte[] content) {
 		this.content = content;
 	}
+	
+	@Override
+	public Object getContentObject() {
+        MarshallerReaderContext context = null;
+        try {
+            ByteArrayInputStream stream = new ByteArrayInputStream(content);
```

Comment:
this can't be here as it breaks functionality to deal with variables that need to use kjar class loader of custom marshalling strategies - e.g. JPA one. Moreover this does not seem like a good place to be included in the entity, any reason for this?